pallets/crowdloans/src/lib.rs

```diff
@@ -199,7 +199,7 @@ pub mod pallet {
         AllMigrated(ParaId),
         /// Partially contributions migrated
         /// [para_id, non_migrated_count]
-        PartiallyMigrated(ParaId, VaultId),
+        PartiallyMigrated(ParaId, u32),
```

Comment:
Can we use `TrieIndex` instead of u32

Reply:
@ashi20192020ayush it's for `non_migrated_count` so better be u32